### DISCUSSION TITLE
unable to start nginx error

### DISCUSSION THREAD

**Discussion by devopsmins** [learndevopsonline/learndevopsonline#1671] [Q&A]
![nginx](https://github.com/learndevopsonline/learndevopsonline/assets/150717333/7a11dbf1-3de0-46e2-a797-883a43041090)

**Comment by sreedharm07** [comment]
@devopsmins 
epecting semi-colon here
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/06dbd9b1-8c0c-44ff-84b7-458bd3798f6b)

**Reply by devopsmins** [reply]
okay...Thanks
